system-tests/fixtures/ios-container/Podfile.lock

```diff
@@ -15,16 +15,16 @@ PODS:
     - Flipper-Folly (~> 2.1)
     - Flipper-RSocket (~> 1.0)
   - Flipper-DoubleConversion (1.1.7)
-  - Flipper-Folly (2.1.1):
+  - Flipper-Folly (2.2.0):
```

Comment:
Please exclude this file from your PR - I've already put it on the system-test ignore list in #1575 (merge pending).

Reply:
sounds good, thanks

Reply:
i should exclude the proj file as well right?